CONTRIBUTING.md

```diff
@@ -6,6 +6,25 @@ You found something you'd like to change? Great! Please submit a pull request an
 2. Push to the branch (`git push origin my-new-feature`)
 3. Create new Pull Request
 
+## What to contribute
+There are many ways that you can contribute to the Azure Event Hubs client project:
+
+* Submit a bug
+* Submit a code fix for a bug
+* Submit additions or modifications to the documentation
+* Submit a feature request
+
+All code submissions will be reviewed and tested by the team, and those that meet a high bar for both quality and design/roadmap appropriateness will be merged into the source. Be sure to follow the existing file/folder structure when adding new boards or sensors.
+
+This project welcomes contributions and suggestions. Most contributions require you to
```

Comment:
I would put this CLA section at the top of the file.

Reply:
I agree. Both this paragraph and the one after it should move up to the previous section

Reply:
Updated